app/controllers/lectures_controller.rb

```diff
@@ -25,18 +26,24 @@ def edit
 
   # POST /lectures
   def create
-    @lecture = Lecture.new(lecture_params)
-
-    if @lecture.save
-      redirect_to @lecture, notice: "Lecture was successfully created."
+    if current_user.is_student?
+      redirect_to lectures_url, notice: "You are a student! You can not create a lecture :("
```

Comment:
if/else is her not needed, only the if branch.

You can end  it after this line. So the nesting level is decreased and comprehensibility increased. I try to make it visible by suggestions. Unfortunatly I have to use multiple, so If you like the idea accept all and if not decline all.
I'm not sure about the indentation and rubo cop, maybe it is the easiest to edit file manually ;) without my suggestions?!

```suggestion
      redirect_to lectures_url, notice: "You are a student! You can not create a lecture :("
    end
```

Reply:
When adapting your changes I get an error that redirect_to/render was called multiple times. So I don't think this is possible